### PR TITLE
Debugging GH Secrets

### PR DESCRIPTION
**Pull request by aronchick**
Fixing GH to allow for bots to manage PRs.

### CODE REVIEW

File: .circleci/trigger_circleci.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def main():
     }
     data.update(target)
 
+    print(f"Full data: {data}")
```
Comment (frrist) [inline .circleci/trigger_circleci.py:38]: any chance of this leaking secrets?

**Review by frrist — APPROVED** [review]
One comment regarding what we print.

File: .circleci/trigger_circleci.py (hunk 1 above)
Comment (aronchick) [reply .circleci/trigger_circleci.py:38]: no - this is from github event and github auto hides.

but will remove when this is debugged.

**Pull request merged**